platform-includes/getting-started-use/javascript.nestjs.mdx

```diff
@@ -49,7 +49,6 @@ export class YourCatchAllExceptionFilter implements ExceptionFilter {
 }
 ```
 
-
```

Comment:
l: Can we also mention the last point, about HttpExceptions not being caught, below the code block just above this line? I think it's valuable information since users might expect otherwise in a catch-all filter?

Reply:
LGTM as you have it now